### ISSUE TITLE
Document differences between native element and our update, specifically the words used, and decide who this information should be shared with

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-frontend#5625]
## What
Document differences between native element and our update, specifically the words used, and decide who this information should be shared with

Browsers use different words to label the button and the feedback that no file was chosen.
We will only be able to use one label, which means it might be different to what people expect. (Or they might not notice it at all.)

How browsers label elements of the native input:

* Chrome: Choose file, No file chosen
* Firefox: Browse..., No file selected
* Safari: Choose file, No file selected

## Why

To make the component consistent across browsers.

## Done when
- [ ] Research custom file upload components and how Dragon users call them
- [ ] Determine which audience needs to know about this 
- [ ] Document what those users need to know
- [ ] Publish in the channel most relevant to those users

**Comment by CharlotteDowns** [comment]
This task is now capturing what we put into the guidance to let users know why we have added a enhancement and how to activate it.
